sci-libs/blis/blis-0.6.0.ebuild

```diff
@@ -53,7 +57,7 @@ src_configure() {
 		--enable-verbose-make \
 		--prefix="${BROOT}"/usr \
```

Comment:
This is probably ESYSROOT instead as we want DEPEND-clas stuff installed, but I'm also not going to ask you to mess with a fragile BLAS like thing right now, just worth thinking about

Reply:
Okay, thanks for pointing out